bigbluebutton-html5/public/locales/en.json

```
@@ -288,8 +288,8 @@
     "app.presentationUploader.exportingTimeout": "The export is taking too long...",
     "app.presentationUploader.export": "Send to chat",
     "app.presentationUploader.exportCurrentStatePresentation": "Send out a download link for the presentation in its current state",
-    "app.presentationUploader.enableOriginalPresentationDownload": "Enable download of the original presentation",
-    "app.presentationUploader.disableOriginalPresentationDownload": "Disable download of the original presentation",
+    "app.presentationUploader.enableOriginalPresentationDownload": "Enable download of the original presentation ({0})",
+    "app.presentationUploader.disableOriginalPresentationDownload": "Disable download of the original presentation ({0})",
```

```suggestion
    "app.presentationUploader.enableOriginalPresentationDownload": "Enable download of the presentation ({0})",
    "app.presentationUploader.disableOriginalPresentationDownload": "Disable download of the presentation ({0})",
```

I think the "Original" word may make it confusing.. once the Converted file is not the original anymore in my view..